recipes/ad3/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ad3" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/andre-martins/AD3/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e342c1178faf7c232ad4c2df714916a30459606de841de40286ab90f523100f5
+
+build:
+  number: 0
+  skip: True  # [py<37 or py>37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python ==3.7.*
+    - pip
+  run:
+    - python ==3.7.*
```

Comment:
Don't list a Python version dependency. This is already handled by `skip` above.